logrotate.8.in

```diff
@@ -2,89 +2,88 @@
 .SH NAME
 logrotate \(hy rotates, compresses, and mails system logs
 .SH SYNOPSIS
-\fBlogrotate\fR [\fB\-dv\fR] [\fB\-f\fR|\fB\-\-force\fR]
-[\fB\-s\fR|\fB\-\-state \fIfile\fR] \fIconfig_file\fR ..
+\fBlogrotate\fR
+\fR[\fB\-\-debug\fR]
+\fR[\fB\-\-verbose\fR]
+\fR[\fB\-\-log\fR \fIfile\fR]
+\fR[\fB\-\-force\fR]
+\fR[\fB\-\-state\fR \fIfile\fR]
+\fR[\fB\-\-mail\fR \fIcommand\fR]
+\fIconfig_file\fR
+\fR[\fIconfig_file2 ...\fR]
 .SH DESCRIPTION
 \fBlogrotate\fR is designed to ease administration of systems that generate
-large numbers of log files.  It allows automatic rotation, compression, 
+large numbers of log files.  It allows automatic rotation, compression,
 removal, and mailing of log files.  Each log file may be handled daily,
 weekly, monthly, or when it grows too large.
 .P
-Normally, \fBlogrotate\fR is run as a daily cron job.  It will not modify
-a log more than once in one day unless the criterion for that log is
-based on the log's size and \fBlogrotate\fR is being run more than once
-each day, or unless the \fB\-f\fR or \fB\-\-force\fR option is used.
+Normally, \fBlogrotate\fR is run as a daily cron job.  It will not modify a
+log more than once in one day unless the criterion for that log is based on
+the log's size and \fBlogrotate\fR is being run more than once each day, or
+unless the \fB\-f\fR or \fB\-\-force\fR option is used.
```

Comment:
Please do not move the line breaks where not necessary.  It complicates backporting man page patches for older releases of logrotate.